lib/kafka/client.rb

```diff
@@ -473,6 +473,20 @@ def delete_topic(name, timeout: 30)
       @cluster.delete_topic(name, timeout: timeout)
     end
 
+    # Create partitions for a topic.
+    #
+    # @param name [String] the name of the topic.
+    # @param num_partitions [Integer] the number of desired partitions for
+    # the topic
+    # @param timeout [Integer] a duration of time to wait for the new
+    # @param validate_only [Boolean] whether this request is to validate only
+    # without actually execute it
```

Comment:
I don't see any use cases for `validate_only` in ruby-kafka – do you have any? Otherwise I think it's better to leave it out (and just pass `false`).